packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -26,6 +27,24 @@ export const metadata: ConnectorMetadata = {
   },
 };
 
+/**
+ * UsageType here is used to specify the use case of the template, can be either
+ * 'Register', 'SignIn', 'ForgotPassword' or 'Test'.
+ */
+const templateGuard = z.object({
+  usageType: z.nativeEnum(PasscodeType),
```

Comment:
will it be better to define a TemplateType in `../types` to make it clear? for now it can be just PasscodeType.

```ts
type TemplateType = PasscodeType;
```

Reply:
year, for now, only passcode type, but in the future, will have more types, SmsConnector is not just for "passcode".

For now, is fine to take no action.

Reply:
sorry, after further reading, I think is bettor to take @gao-sun's suggestion right now.

```
type TemplateType = PasscodeType | 'Test';
```

is a bad idea to add `Test` directly to `PasscodeType`